storage/client_test.go

```diff
@@ -1385,9 +1412,44 @@ func TestRetryNeverEmulated(t *testing.T) {
 				t.Errorf("GetBucket: got unexpected error %v; want 503", err)
 			}
 		}
+		// Error should be wrapped so it indicates that MaxAttempts has been reached.
+		if got, want := err.Error(), "retry failed after 3 attempts"; !strings.Contains(got, want) {
+			t.Errorf("got error: %q, want to contain: %q", got, want)
+		}
```

Comment:
Do we need to delete the retry test resource in the testbench after finishing?

Reply:
I don't think it matters too much given that the emulator is torn down between test runs, but on the other hand I might as well -- will update.